crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -145,6 +148,7 @@ impl C8yMapperConfig {
             capabilities,
             auth_proxy_addr,
             auth_proxy_port,
+            tedge_config.mqtt.topic_root.clone(),
```

Comment:
Since the schema is created above anyway, why not just pass it here as well? Else why not just directly pass `tedge_config.mqtt.topic_root.clone()` as the argument?

Reply:
yes, it makes sense. I fixed it.